back/src/cmd/app/main.go

```diff
@@ -32,6 +32,10 @@ func serve() {
 	if frontHost := os.Getenv("FRONT_HOST"); frontHost != "" {
 		r.Use(cors.New(cors.Config{
 			AllowOrigins: []string{frontHost},
+			AllowHeaders: []string{
+				"Access-Control-Allow-Headers",
+				"Content-Type",
+			},
```

Comment:
ありがとうございます。
問題ありませんでした！

今後のユーザの更新・削除 (PUT, DELETE) に備えて、
`Access-Control-Allow-Headers` も追加するとより良いかもしれません ...


```suggestion
			},
			AllowMethods: []string{"PUT", "DELETE"},
```

(こちらがないと、更新・削除では cors でエラーしてしまう印象があります ...)

ご検討いただけると幸いです 🙇